src/ctapipe/calib/camera/calibrator.py

```diff
@@ -211,24 +215,27 @@ def _calibrate_dl1(self, event, tel_id):
         if self._check_dl0_empty(waveforms):
             return
 
-        n_pixels, n_samples = waveforms.shape
+        n_channels, n_pixels, n_samples = waveforms.shape
 
         selected_gain_channel = event.dl0.tel[tel_id].selected_gain_channel
         broken_pixels = _get_invalid_pixels(
+            n_channels,
             n_pixels,
             event.mon.tel[tel_id].pixel_status,
             selected_gain_channel,
         )
+        pixel_index = np.arange(n_pixels)
 
         dl1_calib = event.calibration.tel[tel_id].dl1
-        time_shift = event.calibration.tel[tel_id].dl1.time_shift
+        ped_offset = dl1_calib.pedestal_offset
+        time_shift = dl1_calib.time_shift
```

Comment:
I'd stay consistent on how to access the data of the dl1 calibration container, i.e. always use `dl1_calib.pedestal_offset` and `dl1_calib.time_shift` as you do it later the relative and absolute factor.

Reply:
Using the full name everywhere is running into line length limitations of the formatter att several points and results in awkward multi-line statements.

I think the current version is more readable.